packages/utils/lib/express/route.ts

```diff
@@ -0,0 +1,85 @@
+import type { Request, Response, NextFunction, Express } from 'express';
+import type { Endpoint, EndpointDefinition } from '@nangohq/types';
+
+export type EndpointRequest<E extends EndpointDefinition> = Request<
+    Endpoint<E>['Params'],
+    Endpoint<E>['Reply'],
+    Endpoint<E>['Body'],
+    Endpoint<E>['Querystring']
+>;
+export type EndpointResponse<E extends EndpointDefinition> = Response<Endpoint<E>['Reply']>;
+
+export interface Route<E extends EndpointDefinition> {
+    path: E['Path'];
+    method: E['Method'];
+}
+
+export interface RouteHandler<E extends EndpointDefinition> extends Route<E> {
+    validate: (req: EndpointRequest<E>, res: EndpointResponse<E>, next: NextFunction) => void;
+    handler: (req: EndpointRequest<E>, res: EndpointResponse<E>, next: NextFunction) => void;
+}
+
+export const createRoute = <E extends EndpointDefinition>(server: Express, rh: RouteHandler<E>): void => {
+    if (rh.method === 'GET') {
+        server.get(rh.path, rh.validate, rh.handler);
+    } else if (rh.method === 'POST') {
+        server.post(rh.path, rh.validate, rh.handler);
+    } else if (rh.method === 'PATCH') {
+        server.patch(rh.path, rh.validate, rh.handler);
+    } else if (rh.method === 'PUT') {
+        server.put(rh.path, rh.validate, rh.handler);
+    } else if (rh.method === 'DELETE') {
+        server.delete(rh.path, rh.validate, rh.handler);
+    }
```

Comment:
rh.handler is not wrapped in any await try/catch. Any throw will crash the process